kernel/arch/dreamcast/kernel/irq.c

```diff
@@ -83,22 +83,71 @@ int trapa_set_handler(irq_t code, irq_handler hnd, void *data) {
     return 0;
 }
 
+/* Get a string description of the exception */
+static char *irq_exception_string(int evt) {
+    switch(evt) {
+        case EXC_ILLEGAL_INSTR:
+            return "Illegal instruction";
+        case EXC_SLOT_ILLEGAL_INSTR:
+            return "Slot illegal instruction";
+        case EXC_GENERAL_FPU:
+            return "General FPU exception";
+        case EXC_SLOT_FPU:
+            return "Slot FPU exception";
+        case EXC_DATA_ADDRESS_READ:
+            return "Data address error (read)";
+        case EXC_DATA_ADDRESS_WRITE:
+            return "Data address error (write)";
+        case EXC_DTLB_MISS_READ:
+            return "Data TLB miss (read)";
+        case EXC_DTLB_MISS_WRITE:
+            return "Data TLB miss (write)";
+        case EXC_DTLB_PV_READ:
+            return "Data TLB protection violation (read)";
```

Comment:
```suggestion
        case EXC_ITLB_MISS:
        case EXC_DTLB_MISS_READ:
            return "Instruction or Data(read) TLB miss";
        case EXC_DTLB_MISS_WRITE:
            return "Data(write) TLB miss";
        case EXC_ITLB_PV:            
        case EXC_DTLB_PV_READ:
            return "Instruction or Data(read) TLB protection violation";
```
There are a few of the codes that have two exceptions mapped to them. Or contextually do the iTLB codes not make sense to include?